docs/index.bs

```diff
@@ -3057,7 +3057,10 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                       1. If |navigationPreloadResponse|'s [=response/type=] is "`error`", reject |preloadResponse| with a `TypeError` and terminate these substeps.
                       1. Associate |preloadResponseObject| with |navigationPreloadResponse|.
                       1. Resolve |preloadResponse| with |preloadResponseObject|.
-              1. [=If aborted=], then [=fetch controller/abort=] |preloadFetchController|.
+              1. [=If aborted=], then:
+                  1. If |controller|'s [=fetch controller/serialized abort reason=] is non-null, let |deserializedError| be the result of calling [$StructuredDeserialize$] with |controller|'s [=fetch controller/serialized abort reason=] and |workerRealm|. If that threw an exception or returns undefined, then let |deserializedError| be a "{{AbortError}}" {{DOMException}}.
+                  1. Otherwise, let |deserializedError| be a "{{AbortError}}" {{DOMException}}.
```

Comment:
I don't think you can do "Otherwise let".
I suggest to have this row before the previous one as the default, and then `if ... set` without an otherwise.